### PR TITLE
Standarize variables names

### PR DESCRIPTION
**Pull request by CristhianMotoche**
I've standarize the variables, please take a look if I'm omiting something. Thanks

@sestrella, @goetzc and @lpaulmp I'll work in this one to solve #48 
I'll be waiting for suggestions or feedback 👍

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -22,39 +22,46 @@ dependencies:
   - src: kamaln7.swapfile
     version: "0.4"
     become: yes
-    remote_user: "{{ admin_user }}"
-    swapfile_size: "{{ swap_file_size }}"
+    become_method: sudo
```
Comment (goetzc) [inline meta/main.yml]: These lines should not be required on that file.

File: meta/main.yml [hunk 2]
```diff
@@ -22,39 +22,46 @@ dependencies:
...
+    remote_user: "{{ sb_debian_base_admin_user }}"
+    unattended_mail: "{{ sb_debian_base_uu_email_alerts }}"
     unattended_automatic_reboot: false
     when: bootstrap is defined
 
   - src: nickjj.fail2ban
     version: v0.1.2
     become: yes
     remote_user: "{{ admin_user }}"
```
Comment (goetzc) [inline meta/main.yml]: This is duplicated

File: tasks/deploy-username-tasks.yml [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
-    name: "{{ deploy_user_group }}"
+    name: "{{ sb_debian_base_deploy_user_group }}"
     state: present
 
 - name: Create deploy user
   user:
-    name: "{{ deploy_username }}"
-    comment: "Unprivileged {{ deploy_username }} user"
-    group: "{{ deploy_user_group }}"
+    name: "{{ sb_debian_base_deploy_user }}" comment: "Unprivileged {{ sb_debian_base_deploy_user }} User"
```
Comment (goetzc) [inline tasks/deploy-username-tasks.yml]: I think the rebase is not working as should :D

Comment (CristhianMotoche) [inline tasks/deploy-username-tasks.yml]: Sorry @goetzc 
I'll fix it!

File: meta/main.yml (hunk 1 above)
Comment (CristhianMotoche) [inline meta/main.yml]: All the `become_method: sudo`

**Comment by goetzc** [comment]
LGTM! @CristhianMotoche

**Pull request merged**